rest/route/admin_flags_test.go

```diff
@@ -0,0 +1,69 @@
+package route
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"golang.org/x/net/context"
+)
+
+type AdminFlagsRouteSuite struct {
```

Comment:
again, maybe it makes sense to make this into a single case rather than a suite?

Reply:
Done